packages/devtools_app/lib/src/screens/logging/logging_controller.dart

```diff
@@ -511,11 +511,9 @@ class LoggingController extends DisposableController
       return node;
     }
     RemoteDiagnosticsNode? summary;
-    if (node.inlineProperties != null) {
-      for (RemoteDiagnosticsNode property in node.inlineProperties!) {
-        summary = _findFirstSummary(property);
-        if (summary != null) return summary;
-      }
+    for (RemoteDiagnosticsNode property in node.inlineProperties) {
```

Comment:
prefer var instead of explicit type in loop

Reply:
done